doc: Doc fixes for 2.8 (#18595)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_qsg.rst

   CONFIG_SUIT_MPI_APP_LOCAL_1_VENDOR_NAME="ACME Corp"
   CONFIG_SUIT_MPI_APP_LOCAL_1_CLASS_NAME="Light bulb"

> After rebuilding you will be able to find the following part in the :file:`./build/DFU/application.yaml` file.

Reviewer: ```suggestion
After rebuilding you can find the following part in the :file:`./build/DFU/application.yaml` file.
```
